src/app/templates/index.ftl

```diff
@@ -149,6 +149,11 @@
                 "${pinnedReport.name}",
                 </#list>];
             </#if>
+            var runReportBtn = document.querySelector('#run-report')
+            runReportBtn.addEventListener('click', () => {
+                window.sessionStorage.removeItem("selectedRunningReportTab");
+                window.sessionStorage.removeItem("selectedRunningReportKey");
```

Comment:
Obviously this clears selected key whenever selected tab is also cleared. This might be unnecessary as the key will be changed again if a user actually runs a workflow. But doesn't so any harm - hard to predict what the user would expect here.

Reply:
I think it would be nice to retain the key - I think that should be mean that if they immediately clicked onto the logs tab having browsed back to the run report page, they would see the logs for the last selected report. It's right that the link should always open on the run tab, but if someone did want to revisit their last selected logs this would make it easier for them, and as you say if they actually ran a new report, that would become the new selected key.

Reply:
Ok, I'll stop clearing the running key from session in each of the links where I clear the tab.